doc/source/user_guide/fields/field_data.rst

```diff
@@ -27,25 +27,31 @@ You can do this either by loading both case and data files or by reading a case
 Simple requests
 ---------------
 
-Here are the methods for requesting each type of field:
+For creating a request you need to construct a request object of type:
```

Comment:
@prmukherj Once again, please check the above sentence. Otherwise, we are creating documentation that will need to be reworded later.